iis-wf/iis-wf-export-actionmanager/src/main/java/eu/dnetlib/iis/wf/export/actionmanager/api/ActionManagerServiceFacade.java

```diff
@@ -20,19 +22,8 @@
 	 * @param provenance
 	 * @param trust
 	 * @param nsprefix
-	 * @return number ofactions successfully stored
 	 * @throws ActionManagerException
```

Comment:
The params in this comment are not explained. One can see what params the method takes without this comment, but one can't know what they are.